src/main/controllers/helpers/TribunalOrderOrRequestHelper.ts

```diff
@@ -199,21 +199,47 @@ export const populateNotificationsWithRedirectLinksAndStatusColors = (
 };
 
 export const activateTribunalOrdersAndRequestsLink = (items: SendNotificationTypeItem[], req: AppRequest): void => {
+  if (!items?.length) {
+    return;
+  }
+  const notices = filterNotificationsWithRequestsOrOrders(items).filter(
+    notice => notice.value.sendNotificationNotify !== Parties.RESPONDENT_ONLY
+  );
+  if (!notices?.length) {
+    return;
+  }
+
   const userCase = req.session?.userCase;
-  if (items?.length && filterNotificationsWithRequestsOrOrders(items).length) {
-    const respOnly = items.some(
-      item =>
-        item.value.sendNotificationSelectParties === Parties.RESPONDENT_ONLY &&
-        item.value.sendNotificationResponseTribunal === ResponseRequired.YES
+  const anyRequireResponse = notices.some(
+    item =>
+      item.value.sendNotificationResponseTribunal === ResponseRequired.YES &&
+      item.value.sendNotificationSelectParties !== Parties.RESPONDENT_ONLY
+  );
+  const anyRequireResponseAndNotResponded = notices.some(item => {
+    return (
+      item.value.sendNotificationResponseTribunal === ResponseRequired.YES &&
```

Comment:
This two lines are duplicated above and can be moved to a const?

Reply:
done.